src/OpenSage.Game/Data/Ini/BitArray`1.cs

```diff
@@ -112,6 +112,12 @@ public override int GetHashCode()
             return HashCode.Combine(_inner.Count);
         }
 
+        public BitArray<TEnum> Clone()
```

Comment:
I think you could inherit from ICloneable

Reply:
Fixed